app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,16 @@
             </intent-filter>
         </activity>
 
+
+        <activity android:name=".PermissionActivity"
+            android:exported="true">
+            <intent-filter>
+                <action android:name="custom.intent.permission.request" />
```

Comment:
we should rename the intent action here to - `org.commcare.dalvik.reminders.permission`

Reply:
Done